src/main/msp/msp.c

```diff
@@ -3280,15 +3281,39 @@ static mspResult_e mspCommonProcessInCommand(mspDescriptor_t srcDesc, uint8_t cm
         break;
 
     case MSP_OSD_CHAR_WRITE:
-#ifdef USE_MAX7456
+#if defined(USE_OSD)
```

Comment:
lol, this is all already inside a giant `#if defined(USE_OSD)`: https://github.com/betaflight/betaflight/pull/9127/files#diff-eefd576c86ccfa8b6cdf98081806a7a8R3195